src/shacl2code/model.py

```diff
@@ -84,6 +84,7 @@ class Class:
     id_property: str = ""
     node_kind: str = None
     is_extensible: bool = False
+    is_abstract: bool = False
```

Comment:
We should pull out the abstract handling to a different PR (and we can merge it before go support). I can do that if you want

Reply:
Let me figure out if I can make all the classes interfaces. In this case, I won't need this.